subt/docker/robotika/run_solution.bash

```diff
@@ -24,9 +20,14 @@ while [ -z "$ROBOT_NAME" ]; do
 done
 echo "Robot name is '$ROBOT_NAME'"
 
+echo "Start robot solution"
+export OSGAR_LOGS=/osgar-ws/logs
+/osgar-ws/env/bin/python3 -m subt run /osgar-ws/src/subt/zmq-subt-x2.json --side auto --walldist 0.8 --timeout 100 --speed 1.0 --note "run_solution.bash" &
```

Comment:
do we really need our own python if we can take over the whole docker image?

Reply:
Unfortunately yes, we do (it is only virtual environment). I tried taking over the system python but actually ROS was having problems then.